R/frontline_dm.R

```diff
@@ -44,7 +57,7 @@ col.mod_date <- function(df,
                                      origin = "1970-01-01"),
                   days_since_mod = max_date - mod_date) %>% 
     dplyr::select(-last_modified_at_a) %>% 
-    dplyr::filter(days_since_mod >= 0 & days_since_mod <= 14) %>% 
+    dplyr::filter(days_since_mod >= 0 & days_since_mod <= 15) %>% 
     dplyr::group_by(operation_number) %>% 
```

Comment:
tweak window of included responses so it's the same as the nov twc enrollment report